node_modules/col-whiteboards/tests/util.js

```diff
@@ -99,16 +124,37 @@ var assertCreateWhiteboard = module.exports.assertCreateWhiteboard = function(cl
       assert.strictEqual(whiteboard.title, title);
 
       // Verify that the expected members are present
-      var expectedMembers = CollabosphereUtil.toArray(members);
-      expectedMembers.push(me.id);
-      expectedMembers = _.uniq(expectedMembers);
-
       assert.strictEqual(whiteboard.members.length, expectedMembers.length);
       _.each(expectedMembers, function(member) {
         UsersTestUtil.assertUser(_.findWhere(whiteboard.members, {'id': member}));
       });
 
-      return callback(whiteboard);
+      _whiteboard = whiteboard;
+      return done();
+    });
+
+    // Require the whiteboards API inline as it requires the entire application to be bootstrapped
+    // when loaded the first time
+    var WhiteboardsAPI = require('../lib/api');
+    WhiteboardsAPI.once(WhiteboardsAPI.EVENT_NAMES.INVITATIONS_SENT, function() {
```

Comment:
It's hard to test that in the utility in a straightforward way, without leaking event listeners.

I'll add a test that:
 - attaches an event listener through `on` on the`EmailUtil` that expects an email for a user *once* and throws an error if there's more
 - create a whiteboard and add 1 user
 - edit a whiteboard and add 1 user